src/main/java/com/artipie/management/api/ApiRepoPostRtSlice.java

```diff
@@ -65,23 +64,26 @@ public Response response(
                 .thenApply(
                     form -> {
                         final Response res;
-                        final String action = ApiRepoPostRtSlice.value(form, "action");
-                        if (Action.UPDATE.value().equals(action)) {
+                        final ValueFromBody vals = new ValueFromBody(form);
+                        final Optional<String> meth = vals.byName("method");
+                        if (meth.isPresent() && Action.UPDATE.value().equals(meth.get())) {
                             res = new ApiRepoUpdateSlice(this.configfile)
                                 .response(
-                                    line,
-                                    headers,
-                                    new Content.From(form.getBytes(StandardCharsets.US_ASCII))
+                                    line, headers,
+                                    new Content.From(
+                                        vals.payload().getBytes(StandardCharsets.US_ASCII)
```

Comment:
Are you sure we always use ASCII for configurayion? E.g. can we name some repository or user with unicode symbols? If we can, then we need to decode it with UTF-8.